datadog/dogshell/monitor.py

```diff
@@ -128,7 +128,7 @@ def setup_parser(cls, subparsers):
                                                   help="Validates if a monitor definition is correct")
         validate_parser.add_argument('type', help="type of the monitor, e.g."
                                      "'metric alert' 'service check'")
-        validate_parser.add_argument('query', help="query to notify on with syntax varying "
+        validate_parser.add_argument('query', help="the monitor query"
                                      "depending on what type of monitor you are creating")
```

Comment:
Now the sentence doesn't make much sense. Let's do something like `monitor query, the syntax varies depending on the type of the monitor`